src/common/parseDataUtil.js

```diff
@@ -1,32 +1,30 @@
 import _ from 'lodash';
 import moment from 'moment';
-import { blockHeightKeyInfos } from './constants';
+import { blockHeightKeyInfos, TxStatus } from './constants';
 
 const parseDataUtil = {
   /**
    * get transaction from parse transaction object
    * @param {*} txOjbect, parse transaction object
    */
-  getTransaction(txOjbect) {
-    const createdAt = txOjbect.get('createdAt');
-    const confirmedAt = txOjbect.get('confirmedAt');
-    const receivedAt = txOjbect.get('receivedAt');
+  getTransaction(txObject) {
+    const status = txObject.get('status');
+    let dateTime = status === TxStatus.SUCCESS ? txObject.get('confirmedAt') : txObject.get('createdAt');
+    dateTime = dateTime ? moment(dateTime) : moment(0);
```

Comment:
No you can't use moment(0). That's year 1970. Left it undefined here, and don't show in UI.